packages/config/index.js

```diff
@@ -0,0 +1,52 @@
+export const { NODE_ENV } = process.env
+export const IPFS_GATEWAY = 'https://cloudflare-ipfs.com'
+export const ARWEAVE_GATEWAY = 'https://arweave.net'
+export const API_WALLET_ADDRESS = process.env.API_WALLET_ADDRESS || 'like1sdz7wpzdazxekr6f4sx33rtc7nra6lu9nxe7yh'
+export const EXPIRATION = 30 // 30 days
+export const UI_BASE_PATH = '';
+const UI_BASE = process.env.UI_BASE || 'http://localhost:3000'
+export const UI_URL = `${UI_BASE}${UI_BASE_PATH}`;
+export const BACKEND_PORT = 8000;
+export const BACKEND_URL = process.env.BACKEND_URL || `http://127.0.0.1:${BACKEND_PORT}`
+export const isDev = NODE_ENV === 'development'
+export const ENDPOINT = isDev
+  ? 'https://node.testnet.like.co'
+  : 'https://mainnet-node.like.co'
+
+export const WIDGET_ENDPOINT = isDev
+  ? 'https://testnet.like.co'
+  : 'https://like.co'
+
+export const WALLET_CONFIG = isDev
```

Comment:
Usually, use `IS_TESTNET` to distinguish if the `NODE_ENV` is `'production'`.

```suggestion
export const IS_TESTNET = NODE_ENV !== 'production'
export const ENDPOINT = IS_TESTNET
  ? 'https://node.testnet.like.co'
  : 'https://mainnet-node.like.co'

export const WIDGET_ENDPOINT = IS_TESTNET
  ? 'https://testnet.like.co'
  : 'https://like.co'

export const WALLET_CONFIG = IS_TESTNET
```